services/eventhistory/README.md

```diff
@@ -0,0 +1,27 @@
+# Eventhistory Service
+
+The `eventhistory` consumes all events from the configured event system like NATS, stores them and allows to retrieve them via an eventid.
+
+## Prerequisites
+
+Running the eventhistory service without an event sytem like NATS is not possible.
+
+## Consuming
+
+The `eventhistory` services consumes all events from the configured event sytem.
+
+## Storing
+
+The `eventhistory` stores each consumed event in the configured store. Possible stores are:
+  - `mem`: Basic inmemory store and the default.
+  - `ocmem`: Advanced inmemory store allowing max size.
+  - `redis`: Stores data in a configured redis cluster.
+  - `etcd`: Stores data in a configured etcd cluster.
+  - `nats-js`: Stores data using nats.
```

Comment:
Though I heard about redis and etcd, I never heard of `nats-js`. Maybe to add a link for more reading?

Reply:
It's `nats jetstream`. The same technology we are using as eventbus. It also provides key-value store functionality. I added a link.